testnets/_IBC/prysm-celestia.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../ibc_data.schema.json",
```

Comment:
The schema reference must point to the file, which is two directories back (i.e., "../../ibc_data.schema.json")